ci/init-build.sh

```diff
@@ -18,8 +18,15 @@ echo -e "User: ${USER}"
 echo -e "User HOME directory: ${HOME}"
 echo -e "************************************"
 
+echo -e "Checking for sudo, not available on Windows bash"
+SUDO=
+type sudo &> /dev/null
+if [ $? -eq 0 ]; then
+  SUDO=sudo
+fi
+
 echo -e "Setting build environment...\n"
-sudo mkdir -p /etc/cas/config /etc/cas/saml /etc/cas/services
+$SUDO mkdir -p /etc/cas/config /etc/cas/saml /etc/cas/services /tmp /temp
```

Comment:
This makes this script work on Windows git bash (without an error about sudo not existing). Since it is using Windows "Git Bash" (rather than msys2 which is also on the github CI windows image), the home directory is the same as it would be in powershell or cmd.exe (so the `gradle.properties` being created is correct), but when it is making these directories, they are in the Git Bash root folder which is not easily accessible in same path on C or D drive in powershell. The build and tests eventually run in powershell b/c that is default shell on windows, although we could specify that they run in bash. To make folders on C: or D: drives it would need to prefix all the paths with /c or /d but I don't know how to get the current drive letter in Git Bash (doesn't have that concept like powershell does) and its not really important because these folders aren't currently required by the tests (on Windows).